documentation/anonymization.markdown

```diff
@@ -63,6 +63,8 @@ The `moderate` profile attempts to be a bit smarter about what it anonymizes and
 * report log messages: are always anonymized
 * file names: are always anonymized
 * file numbers: are left as they are
+* metrics: are left intact
```

Comment:
Well this isn't true, the `time` category gets anonymized based on the resource type policy above.